scripts/release.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Did you make sure that this file have the [executable (+x) git flag](https://git-scm.com/docs/git-update-index#git-update-index---chmod-x)?

Reply:
Yes I did!